cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -98,7 +98,7 @@
       cronjob at:'0 10 * * *', do:deploy_dir('bin', 'cron', 'redshift_rollups')
       cronjob at:'1 7 * * 6', do:deploy_dir('bin', 'cron', 'cleanup_workshop_attendance_codes')
       cronjob at:'31 17 * * 1-5', do:deploy_dir('bin', 'cron', 'zendesk_slack_report')
-      cronjob at:'0 12 * * *', do:deploy_dir('bin', 'cron', 'applab_datasets', 'daily_weather')
+      cronjob at:'5 12 * * *', do:deploy_dir('bin', 'cron', 'applab_datasets', 'daily_weather')
```

Comment:
do we still want to run this at 11:05pm EST instead of 12:05am so that all US timezones are on the same day?

Reply:
So turns out I was wrong about that part too haha. This is 12:05am UTC (4:05am PST), so it'll already be all on the same day for the timezones we care about (Hawaii through EST)